js/ui/popup/ui.popup.js

```diff
@@ -661,6 +741,8 @@ const Popup = Overlay.inherit({
     },
 
     _optionChanged: function(args) {
```

Comment:
lets use destructurization here and get rid of `args` using in this function

Reply:
looks like we shouldn't do this, because we pass `args` to the `callBase()` function